src/wam2layers/preprocessing/era5.py

```diff
@@ -304,8 +361,8 @@ def prep_experiment(config_file):
             upper_layer = ~lower_layer
 
         if config.level_type == "pressure_levels":
-            upper_layer = p < pb[:, None, :, :]
-            lower_layer = pb[:, None, :, :] < p
+            upper_layer = p < pb[None, :, :]
+            lower_layer = pb[None, :, :] < p
```

Comment:
Would [`broadcast_like`](https://docs.xarray.dev/en/stable/generated/xarray.DataArray.broadcast_like.html) be a way to make this independent of the total dimensionality?

Reply:
Yes, but I did have to make `pb` a DataArray. But that works.